src/GameObject.h

```diff
@@ -72,6 +74,7 @@ class GameObject {
 
 	// TODO(rgarmsen2295): Move this logic to be done via message passing or similar...
 	void changeShader(const std::string& newShaderName);
+	std::string getShader();

```

Comment:
Should this have an implementation? And related, if this does, then is getting the whole render component necessary?